modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/index.js

```diff
@@ -28,11 +33,59 @@ const InvitesModal = ({observer, onClose, project}) => {
 };
 
 const TeamMembers = () => {
-	const {project} = useOutletContext();
+	const [dxpCloudEnvironment, setDxpCloudEnvironment] = useState();
+	const activationStatusAC = 'Active';
+	const groupId = 'groupid';
+	const {project, subscriptionGroups} = useOutletContext();
 	const [visible, setVisible] = useState(false);
 	const modalProps = useModal({
 		onClose: () => setVisible(false),
 	});
+	const [activatedStatusDXPC, setActivatedStatusDXPC] = useState();
+	const [activatedStatusAC, setActivatedStatusAC] = useState();
+	const [activatedLinkDXPC, setActivatedLinkDXPC] = useState();
+	const [activatedLinkAC, setActivatedLinkAC] = useState();
+
+	useEffect(() => {
+		const getOnboardingFormData = async () => {
+			const {data} = await client.query({
+				query: getDXPCloudEnvironment,
+				variables: {
+					filter: `accountKey eq '${project.accountKey}'`,
+					scopeKey: Liferay.ThemeDisplay.getScopeGroupId(),
+				},
+			});
+
+			if (data) {
+				const dxpProjectId =
+					data.c?.dXPCloudEnvironments?.items[0]?.projectId;
+
+				setDxpCloudEnvironment(dxpProjectId);
+			}
+		};
+		getOnboardingFormData();
+
+		const accountSubscriptionFromDXPC = subscriptionGroups.find(
+			(subscriptionGroup) =>
+				subscriptionGroup.name === PRODUCT_TYPES.dxpCloud
+		);
+		const activationStatusDXPC =
+			accountSubscriptionFromDXPC?.activationStatus;
+
+		if (activationStatusDXPC === STATUS_TAG_TYPE_NAMES.active) {
+			setActivatedStatusDXPC(activationStatusDXPC);
+		}
+		if (activationStatusAC === STATUS_TAG_TYPE_NAMES.active) {
+			setActivatedStatusAC(activationStatusAC);
+		}
+
+		setActivatedLinkDXPC(
+			`https://console.liferay.cloud/projects/${dxpCloudEnvironment}/overview`
+		);
```

Comment:
I think the "activatedLinkDXPC" state it`s not necessary. You can declare a constant to build it.

Just like this in the component:
`const activatedLinkDXPC = https://console.liferay.cloud/projects/${dxpCloudEnvironment}/overview`

Reply:
I need it, because I will sent that to `<ManageProductUser />` component. If that was `const` I have no way to pass this through the `return`.